### PR TITLE
File name and use latest

### PR DESCRIPTION
**Pull request by diemol**

### CODE REVIEW

File: src/main/java/de/zalando/tip/zalenium/proxy/SauceLabsRemoteProxy.java [hunk 1]
```diff
@@ -20,6 +22,7 @@
 import javax.servlet.http.HttpServletResponse;
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.util.HashMap;
```
Comment (diemol) [inline src/main/java/de/zalando/tip/zalenium/proxy/SauceLabsRemoteProxy.java]: MINOR Remove this unused import 'java.util.HashMap'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by diemol** [comment]
SonarQube analysis reported 3 issues
* CRITICAL 1 critical
* MINOR 2 minor

Watch the comments in this conversation to review them.

File: src/main/java/de/zalando/tip/zalenium/proxy/SauceLabsRemoteProxy.java [hunk 2]
```diff
@@ -31,21 +32,18 @@
 
 public class SauceLabsRemoteProxy extends DefaultRemoteProxy {
```
Comment (diemol) [inline src/main/java/de/zalando/tip/zalenium/proxy/SauceLabsRemoteProxy.java]: MINOR Override this superclass' "equals" method. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS2160)

**Comment by diemol** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: src/main/java/de/zalando/tip/zalenium/proxy/SauceLabsRemoteProxy.java [hunk 3]
```diff
@@ -191,4 +175,9 @@ public URL getRemoteHost() {
         return null;
     }
 
+    @Override
+    public boolean equals(Object obj) {
```
Comment (diemol) [inline src/main/java/de/zalando/tip/zalenium/proxy/SauceLabsRemoteProxy.java]: MINOR This class overrides "equals()" and should therefore also override "hashCode()". [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1206)
MINOR Remove this method to simply inherit it. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1185)

**Comment by diemol** [comment]
SonarQube analysis reported 2 issues
* MINOR 2 minor

Watch the comments in this conversation to review them.

File: src/main/java/de/zalando/tip/zalenium/proxy/SauceLabsRemoteProxy.java (hunk 2 above)
Comment (diemol) [inline src/main/java/de/zalando/tip/zalenium/proxy/SauceLabsRemoteProxy.java]: MINOR Override this superclass' "equals" method. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS2160)

**Comment by diemol** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by ryudice** [comment]
Just saw in this line that it needs to be send by the driver https://github.com/zalando-incubator/zalenium/pull/36/files#diff-75763b40629c370aa2c8dbe61d2abf87R121, I'll check my driver.

**Comment by elgalu** [comment]
Yes, works the same way as with Sauce Labs or BrowserStack `name` capability, e.g.

```js
capabilities: {
    browserName: 'chrome',
    name: 'my-suite-login'
}
```

**Pull request merged**